nautobot_data_validation_engine/api/urls.py

```diff
@@ -5,8 +5,7 @@
 from nautobot_data_validation_engine.api import views
 
 
-router = OrderedDefaultRouter()
-router.APIRootView = views.DataValidationEngineRootView
+router = OrderedDefaultRouter(view_name="Data Validation Engine")
```

Comment:
Double-check me on this, but I think this will only work with Nautobot 2.1.9 or later. Might need to make that the new minimum supported version?

Reply:
Yeah, looks that way from the CI failure. :(

Reply:
Yep - was just checking that and you're correct. Validated that it only works for nautobot >= 2.1.9.

Will update the min version accordingly

Reply:
@glennmatthews - after bumping the min nautobot version to `^2.1.9`, CI is failing with an erroring unit test `unable to import toml`, and I can't figure out why: https://github.com/nautobot/nautobot-app-data-validation-engine/actions/runs/9196845218/job/25295919027#step:7:1523

Are there any dependency issues with toml that you've seen?

Reply:
Not an issue I've seen before, but some searching found https://github.com/pylint-dev/pylint/issues/7306... maybe try updating the pylint version here and see if it helps?

Reply:
Did the trick!